src/Neo.Extensions/Utility.cs

```diff
@@ -31,7 +31,7 @@ public Logger()
             }
         }
 
-        public static event LogEventHandler Logging;
```

Comment:
Use `Neo.Extensions` in `Neo.Vm` to avoid duplicate `Neo.VM.Utility.StrictUTF8`?